ui/app/router.js

```diff
@@ -15,7 +15,10 @@ Router.map(function() {
       this.route('logout');
       this.mount('open-api-explorer', { path: '/api-explorer' });
       this.route('license');
-      this.route('requests', { path: '/metrics/requests' });
+      this.route('requests', function() {
```

Comment:
if this route is going to land on the metrics dashboard, what do you think about renaming it to `metrics` instead of `requests`? that way we'll have routes for `metrics.http-requets`, `metrics.entities` and so forth -- each of which will correspond to its own metrics page.

Reply:
I thought about that.  My concern or thought was that it might have been originally called "requests" because it aligned with path on the CLI command.  See PR [here](https://github.com/hashicorp/vault/pull/7541).  If that's not an issue, then I agree we should rename it "metrics".

Reply:
nice thinking -- matching the CLI is a great instinct. in this case, since the api endpoint for http requests is to `sys/internal/counters/requests`, and the new endpoint for active service tokens is `sys/internal/counters/tokens`, meaning they are both nested under `sys/internal/counters`, i think it's okay to use `metrics` here, especially since that'll match the page title in the UI.